nautobot/docs/user-guides/git-data-source.md

```diff
@@ -0,0 +1,269 @@
+# Git as a Data Source
+
+The "Git™ as a Data Source" feature was developed to provide the ability to populate existing data, templates, scripts, and much more into Nautobot; while leveraging the benefits that tools such as GitHub and GitLab already provide, including issue tracking, discussions, pipelines, and approvals.
+
+For more technical details on how to use this feature, please see the documentation on [Git Repositories](../models/extras/gitrepository.md).
+
+
+## Supported Providers
+The feature uses the concept of a `provides` field to map a repository to a use case. A list of the supported options is provided below.
+
+### Core Functionality
+|Name|Summary|
+|:--|:--|
+|[Export Templates](../models/extras/exporttemplate.md)|Nautobot allows users to define custom templates that can be used when exporting objects.|
+|[Jobs](../additional-features/jobs.md)|Jobs are a way for users to execute custom logic on demand from within the Nautobot UI. Jobs can interact directly with Nautobot data to accomplish various data creation, modification, and validation tasks.|
+|[Config Contexts](../models/extras/configcontext.md)|Config contexts can be used to provide additional data that you can't natively store in Nautobot.|
+
+### Examples of Plugins Defining Additional Providers
+|Name|Summary|Related Plugin|
+|:--|:--|:--|
+|Backup Configs|Backup configuration data.|[Golden Config](https://github.com/nautobot/nautobot-plugin-golden-config)|
+|Intended Configs|Stores the intended configurations, this grabs Nautobot data and runs through Jinja Templates.|[Golden Config](https://github.com/nautobot/nautobot-plugin-golden-config)|
+|Jinja Templates|Repository that holds Jinja templates to be used to generate intended configs.|[Golden Config](https://github.com/nautobot/nautobot-plugin-golden-config)|
+
+## Repository Details
+
```

Comment:
Same thing here.  What are "Repository Details" ? Why is it important at this point in the article?  Would add an introduction, transition, or move it.